backend/corpora/lambdas/api/v1/curation/collection_uuid/dataset.py

```diff
@@ -0,0 +1,19 @@
+from flask import g
+
+from backend.corpora.api_server.db import dbconnect
+from backend.corpora.common.entities import Dataset
+from backend.corpora.common.utils.http_exceptions import InvalidParametersHTTPException
+from backend.corpora.lambdas.api.v1.dataset import delete_dataset_common
+
+
+@dbconnect
+def delete_dataset(token_info: dict, collection_uuid: str, curator_tag: str = None, dataset_uuid=None):
```

Comment:
```suggestion
def delete_dataset(token_info: dict, collection_uuid: str, curator_tag: str = None, dataset_uuid = None):
```

Reply:
Also we are sadly bouncing back and forth between `dataset_uuid` and `dataset_id` in our server code... there are lots of instances of each so go with whatever you like 😂 😭

Reply:
As long as we are consistent within amodule, I'm ok with it for now.